lib/chemistrykit/cli/cli.rb

```diff
@@ -295,6 +295,16 @@ def rspec_config(config)
               @driver.get(@config.basic_auth.https_url) if @config.basic_auth.https?
             end
 
+            begin
+              @driver.get(@config.basic_auth.https_url + '/business') if @config.basic_auth.https?
+              @driver.execute_script "window.scrollTo(0, document.body.scrollHeight)"
+              sleep 1
```

Comment:
Why do we need the sleep 1?

Reply:
It just takes a second for the popup to come up, if it does. I don't want to make this thing super complex and I'm not worried about the time since it only happens once.